index.html

```diff
@@ -4,7 +4,24 @@
     <title>Prismarine Web Client</title>
     <link rel="stylesheet" href="styles.css">
   </head>
-  <body id="main-body">
+  <body>
+    <div id="hotbar-wrapper">
+      <img id="hotbar-image" src="textures/1.16.4/gui/widgets.png"> <!--NOTE: DELETE THE PUBLIC HERE BEFORE MERGING! THIS IS FOR TESTING W/O WEBPACK !-->
+      <img id="hotbar-highlight" src="textures/1.16.4/gui/widgets.png">
+      <div id="hotbar-items-wrapper">
+        <img class="hotbar-item" id="hotbar-0" src="textures/1.16.4/blocks/stone.png">
```

Comment:
why are items hardcoded here ?

Reply:
It's to test without Webpack. It's replaced first thing upon spawn, before the loading screen is removed, so it shouldn't be an issue.